src/pages/workspaces/workspace/tools/ExportToolModal.js

```diff
@@ -0,0 +1,123 @@
+      toolName: props.methodConfig.name,
+      error: undefined,
+      exported: false
+    }
+  }
+
+  render() {
+    const { exported } = this.state
+
+    return exported ? this.renderPostExport() : this.renderExportForm()
+  }
+
+  renderExportForm() {
+    const { thisWorkspace, onDismiss } = this.props
+    const { selectedWorkspace, toolName, exporting, error } = this.state
+
+    const errors = validate({ toolName }, {
+      toolName: {
+        presence: { allowEmpty: false },
+        format: {
+          pattern: /^[A-Za-z0-9_\-.]*$/,
+          message: 'can only contain letters, numbers, underscores, dashes, and periods'
+        }
+      }
+    })
+
+    return h(Modal, {
+      title: 'Copy to Workspace',
+      onDismiss,
+      okButton: buttonPrimary({
+        disabled: !selectedWorkspace || errors,
+        onClick: () => this.export()
+      }, ['Export'])
+    }, [
+      requiredFormLabel('Destination'),
+      h(WorkspaceSelector, {
+        filter: ({ workspace: { workspaceId }, accessLevel }) => {
+          return thisWorkspace.workspaceId !== workspaceId && Utils.canWrite(accessLevel)
+        },
+        selectedWorkspace,
+        onWorkspaceSelected: ws => this.setState({ selectedWorkspace: ws })
+      }),
+      requiredFormLabel('Name'),
+      validatedInput({
+        error: Utils.summarizeErrors(errors),
+        inputProps: {
+          value: toolName,
+          onChange: e => this.setState({ toolName: e.target.value })
+        }
+      }),
+      exporting && spinnerOverlay,
+      error && h(ErrorView, { error, collapses: false })
+    ])
+  }
+
+  renderPostExport() {
+    const { onDismiss } = this.props
+    const { selectedWorkspace, toolName } = this.state
+
+    return h(Modal, {
+      title: 'Copy to Workspace',
+      onDismiss,
+      cancelText: 'Stay Here',
+      okButton: buttonPrimary({
+        onClick: () => Nav.goToPath('workflow', {
+          namespace: selectedWorkspace.namespace,
+          name: selectedWorkspace.name,
+          workflowNamespace: selectedWorkspace.namespace,
+          workflowName: toolName
+        })
+      }, ['Go to exported tool'])
+    }, [
+      'Successfully exported ',
+      b([toolName]),
+      ' to ',
+      b([selectedWorkspace.name]),
+      '. Do you want to view the exported tool?'
+    ])
+  }
+
+  async export() {
+    const { thisWorkspace, methodConfig, ajax: { Workspaces } } = this.props
+    const { selectedWorkspace, toolName } = this.state
+
+    try {
+      this.setState({ exporting: true })
+      await Workspaces
+        .workspace(thisWorkspace.namespace, thisWorkspace.name)
+        .methodConfig(methodConfig.namespace, methodConfig.name)
+        .copyTo({
+          destConfigNamespace: selectedWorkspace.namespace,
+          destConfigName: toolName,
+          workspaceName: {
+            namespace: selectedWorkspace.namespace,
+            name: selectedWorkspace.name
+          }
+        })
```

Comment:
Probably worth a comment explaining that the method **config** namespace `destConfigNamespace: selectedWorkspace.namespace` is effectively a dead parameter.

The Terra UI & URL display a chimera of the **method namespace** /  **method config name**, which I agree with.

I checked and found this has been the case for the whole history of Terra: https://github.com/DataBiosphere/terra-ui/pull/729/files#diff-802ffc6b9729586f8d6edd54da9f29c65f3d779ff3faaa79ec5445c92e7428ceR110-R117